README.md

```diff
@@ -1,4 +1,4 @@
-[Fixed Data Table 2](https://schrodinger.github.io/fixed-data-table-2/) for React &middot; ![Travic CI status](https://api.travis-ci.org/schrodinger/fixed-data-table-2.svg?branch=master) [![npm version](https://badge.fury.io/js/fixed-data-table-2.svg)](https://www.npmjs.com/package/fixed-data-table-2)
+[Fixed Data Table 2](https://schrodinger.github.io/fixed-data-table-2/) for React &middot; ![CI status](https://github.com/schrodinger/fixed-data-table-2/actions/workflows/build-and-run-tests.yml/badge.svg) &middot; [![npm version](https://badge.fury.io/js/fixed-data-table-2.svg)](https://www.npmjs.com/package/fixed-data-table-2)
```

Comment:
This fixes the CI badge by having it generated from our `build-and-run-tests` workflow run. 

For context, we used travis long time back for CI, but we have replaced it with Github actions.